src/form-snapshot.tsx

```diff
@@ -39,13 +39,13 @@ export const FormSnapshot: React.FC<Props> = ({
   }
 
   return (
-    <div className="flex flex-col gap-3">
-      <div className="leading-normal text-color-text-primary">
+    <div className="flex flex-col gap-4">
+      <h3 className="leading-normal text-color-text-primary text-lg">
```

Comment:
I think this text is only supposed to be big on the widest breakpoint

Reply:
According to the style refinements figma, I think font-size now just a uniform 20px https://www.figma.com/design/BRxaHYt1ytgBgq9xpmQqQj/Savings-Calculator--25?node-id=319-2719&m=dev

Reply:
Oh ok I was looking at the wrong part (the "User Flow" section at the top)